edgelet/aziot-edged/Cargo.toml

```diff
@@ -14,7 +14,7 @@ log = "0.4"
 serde_json = "1"
 sha2 = "0.10"
 serde = "1"
-tokio = { version = "1", features = ["macros", "rt-multi-thread", "signal", "sync", "time"] }
+tokio = { version = "1", features = ["macros", "parking_lot", "rt-multi-thread", "signal", "sync", "time"] }
```

Comment:
`parking_lot` is not strictly necessary here since the crate does not use `tokio::sync::Mutex`. However, keeping it here would prevent accidental use of `tokio::sync::Mutex` without `parking_lot`.